pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -87,6 +89,8 @@ def __init__(self, wires, *, sync=True, c_dtype=np.complex128, shots=None, batch
         super().__init__(wires, c_dtype=c_dtype, shots=shots)
         self._kokkos_state = _kokkos_dtype(self._state.dtype)(self._state)
         self._sync = sync
+        if not LightningKokkos.kokkos_config:
+            LightningKokkos.kokkos_config = kokkos_config_info()
```

Comment:
```suggestion
```

Reply:
See below.

Reply:
Makes sense to call `kokkos_config_info()` after the Kokkos device was initialized, but why do we have to test if the dictionary is empty to set it? Isn't reasonable to assume that the dictionary is empty when we call the device `__init__`?

Reply:
I put this conditional statement here to ensure that `kokkos_config` is only initialized once since it is a class variable  shared by all objects of `LightningKokkos`. If we remove this conditional statement, `kokkos_config` will behave like a instance variable and the `kokkos_config_info` will be called each time when a new object is created, which brings overhead.

Reply:
Hmmm, yea I see. I think this is fine for now, but we should probably redo this later --- I wonder if we have devices with different backends working together (an openmp device concurrently with an AMD GPU device), they will clash here.

Reply:
Is clash from the `Kokkos::print_configuration()`?

Reply:
Thank you for clarifying.